extensions/adobe/experience/primetime-authentication/application.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":   "https://ns.adobe.com/experience/primetime-authentication/application",
```

Comment:
Consider moving this to a "primetime" vs "primetime-*" URLs are not supposed to be used for this level of granularity.

Reply:
This makes sense.
And if other Primetime teams will need to extend or shift the semantics for some fields, we'll have a great negotiation advantage, as we were here first! :)

Reply:
Done